site/search/SearchPanel.tsx

```diff
@@ -482,7 +482,7 @@ export class InstantSearchContainer extends React.Component {
             >
                 <div className="search-panel">
                     <SearchBox
-                        placeholder="Try “Life expectancy”, “Population in extreme poverty”, “Homicide rate”, Biodiversity”…"
+                        placeholder="Try “Life expectancy”, “Population in extreme poverty”, “Homicide rate”, “Biodiversity”…"
```

Comment:
I would make a constant out of this. Too easy to forget replacing it in both places.